ProjectSettings/SceneSwitcherSettings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "baseSearchFolder": "C:/Users/Gursaanj/Documents/GitHub/BForBoss/Assets/_BForBoss"
+}
```

Comment:
Can you put this file into .gitignore

Reply:
But then if we were to change the root folder, how would we make sure that change is pushed for the rest of the team? Don't think it should be a problem if we leave it as is

Reply:
So as of right now, if this changes, it affects everyone else, 
So it doesn't matter if the root folder changes. I'll still be getting `"C:/Users/Gursaanj/Documents` as my path.

Reply:
Wouldn't it make more sense to build this dynamically? ie. store /Assets/_BForBoss in the json and then concatenate it with the project folder? Alternatively if we're going to store the whole path it would probably be better suited for the EditorPrefs

Reply:
Yeah My bad the path should'nt have that.

Actually, I think this should be better for the UserSettings (instead of ProjectSettings) and maybe place that file in the gitignore?

I was thinking of using EditorPref but if we end up having more settings (not sure what exactly) that scaling that with EditorPrefs for each setting instead of just using a json might be a bit much.  

Since the Json is being Lazily initialized anyways it should be fine!

Reply:
Ok nvm Looks like the preferences Folder sets things by `EditorPreferences ` Will just use that